.github/workflows/feedstock_pr_create.yaml

```diff
@@ -0,0 +1,55 @@
+on:
+  workflow_dispatch:
+    inputs:
+      pypi_version:
+        description: 'released PyPI version to use'
+        required: true
+        
+name: Create Feedstock PR
+jobs:
+  create_feedstock_pr:
+    name: Create Feedstock PR
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout inputted version
+        uses: actions/checkout@v3
+        with:
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+          ref: ${{ github.event.inputs.pypi_version }}
+          path: "./featuretools"
+      - name: Pull latest from upstream for user forked feedstock
+        run: |
+          gh auth status
+          gh repo fork conda-forge/featuretools-feedstock --clone
+          gh repo sync machineAYX/featuretools-feedstock --branch main --source conda-forge/featuretools-feedstock --force
+        env:
+          GITHUB_TOKEN: ${{ secrets.AUTO_APPROVE_TOKEN }}
+      - uses: actions/checkout@v3
+        with:
+          repository: machineAYX/featuretools-feedstock
+          ref: main
+          path: "./featuretools-feedstock"
+          fetch-depth: '0'
+      - name: Run Create Feedstock meta YAML
+        id: create-feedstock-meta
+        uses: alteryx/create-feedstock-meta-yaml@v3
+        with:
+          project: "featuretools"
+          pypi_version: ${{ github.event.inputs.pypi_version }}
+          setup_cfg_filepath: "featuretools/setup.cfg"
+          meta_yaml_filepath: "featuretools-feedstock/recipe/meta.yaml"
+          add_to_test_requirements: "graphviz !=2.47.2"
```

Comment:
Just curious why this is needed?

Reply:
What do you mean?
- conda allows you to specify `non-Python` dependencies. This is one of those.
- This was a bad release at some point (I think)
- I couldn't figure out a clean way to determine non-Python dependencies (from setup.cfg).